src/components/dialogs/sensi-parameters-selector.js

```diff
@@ -41,55 +74,738 @@ const EquipmentType = {
+        id: 'sensiPSTs',
+        Field: SensiPSTs,
+        initialValues: paramSensiPSTs,
+    });
+
+    const [sensiNodes, SensiNodesField] = useSensitivityFactors({
+        id: 'sensiNodes',
+        Field: SensiNodes,
+        initialValues: paramSensiNodes,
+    });
 
     const handleClose = () => {
         props.onClose();
     };
 
     const handleStart = () => {
-        props.onStart(
-            selectedVariablesFiltersListUuids,
-            selectedContingencyListUuids,
-            selectedBranchFiltersListUuids
-        );
+        if (inputForm.validate()) {
+            // TODO : remove 'name' property when storing the parameters in config-server
+            // fetch name from uuid when fetching parameters from config-server
+
+            handleSaveSensiConfigurationParam(
+                'resultsThreshold',
+                resultsThreshold
+            );
+            handleSaveSensiConfigurationParam(
+                'sensiInjectionsSet',
+                sensiInjectionsSet
+            );
+            handleSaveSensiConfigurationParam(
+                'sensiInjections',
+                sensiInjections
+            );
+            handleSaveSensiConfigurationParam('sensiHVDCs', sensiHVDCs);
+            handleSaveSensiConfigurationParam('sensiPSTs', sensiPSTs);
+            handleSaveSensiConfigurationParam('sensiNodes', sensiNodes);
+
+            // we provide to the sensitivity analysis service only the checked items in the configuration
+            const sensiConfiguration = {
+                resultsThreshold: resultsThreshold,
+                sensitivityInjectionsSets: sensiInjectionsSet.filter(
+                    (e) => e.checked
+                ),
+                sensitivityInjections: sensiInjections.filter((e) => e.checked),
+                sensitivityHVDCs: sensiHVDCs.filter((e) => e.checked),
+                sensitivityPSTs: sensiPSTs.filter((e) => e.checked),
+                sensitivityNodes: sensiNodes.filter((e) => e.checked),
+            };
+
+            props.onStart(sensiConfiguration);
+        }
     };
 
-    const renderButtons = () => {
-        return (
-            <Grid container spacing={1} item justifyContent={'center'}>
-                {makeButton(handleClose, 'close', false)}
-                {makeButton(handleStart, 'Execute', false)}
-            </Grid>
+    const handleSaveSensiConfigurationParam = useCallback(
+        (paramName, paramValue) => {
+            updateConfigParameter(
+                SENSI_PARAMETER_PREFIX_IN_DATABASE + paramName,
+                JSON.stringify(paramValue)
+            ).catch((errorMessage) => {
+                snackError({
+                    messageTxt: errorMessage,
+                    headerId: 'paramsChangingError',
+                });
+            });
+        },
+        [snackError]
+    );
+
+    function fetchSensiConfigurationParam(paramName) {
+        return fetchConfigParameter(
```

Comment:
Deal with parameter not yet present in database to avoid errors when we parse the result json